Docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpassword
+      MYSQL_DATABASE: todolist
+    volumes:
+      - mysql_data:/var/lib/mysql
+    networks:
+      - todolist_network
+
+  app:
+    build: .
+    depends_on:
+      - db
+    environment:
+      - DATABASE_URL=mysql://root:rootpassword@db/todolist
+    ports:
+      - "8080:8080"
+    networks:
+      - todolist_network
+    volumes:
+      - .:/app
```

Comment:
Mounting the current directory to the container's /app directory can lead to potential security risks and issues with file permissions. Ensure that this is necessary and consider using a more restrictive volume mount if possible.